### ISSUE TITLE
dot notation

### ISSUE THREAD

**Issue by hasanaburayyan** [winglang/wing#1680]

**Comment by hasanaburayyan** [comment]
@staycoolcall911 @eladb Should this be a P1 because of breaking change?

**Comment by staycoolcall911** [comment]
I think we can still support the current API (json.get) alongside the dot notation and avoid a breaking change

**Comment by hasanaburayyan** [comment]
I think that will cause ambiguity IE how to know if `.get` represents a field in the Json or method.

**Comment by staycoolcall911** [comment]
Good point - thanks! We have a few more similar breaking changes as p2/user-story:breaking-change. I'll apply to this issue as well
